### ISSUE TITLE
Should terraform destroy delete user namespaces like dev,staging,test?

### ISSUE THREAD

**Issue by csantanapr** [ibm-garage-cloud/planning#179]
Should terraform destroy delete user namespaces like dev,staging,test?

When running `terraform destroy` it deletes `tools` namespace, but also the other namespaces created with terraform like dev,staging,test.

When running `terraform destroy` I think it should delete tools, and not user workloads namespaces like dev,staging,test
Deleting these can cause grief or worst deleting workloads are being used.

Maybe it can be a seperate helper script to delete the example workload namespaces created by the user.

**Comment by seansund** [comment]
As we discussed, perhaps we shouldn't be creating them in the first place...

**Comment by csantanapr** [comment]
Yeah I agree they are not need it with the new multiuser project workflow
